apps/minifront/src/components/swap/swap-form/simulate-swap-result/traces/trace/index.tsx

```diff
@@ -27,9 +27,12 @@ export const Trace = ({
   metadataByAssetId: Record<string, Metadata>;
 }) => {
   return (
-    <div className='flex w-full items-center justify-between gap-8'>
+    <div className='flex w-full justify-between gap-8'>
       {trace.value.map((value, index) => (
-        <div key={index} className='flex shrink-0 flex-col gap-1'>
+        <div
+          key={index}
+          className='flex min-w-[60px] max-w-[150px] shrink-0 grow basis-auto flex-col items-center gap-1'
+        >
```

Comment:
tightening pill to dynamically shrink and fit within the available space

--------------------

_before_

<img width="689" alt="Screenshot 2024-11-21 at 10 48 58 PM" src="https://github.com/user-attachments/assets/3fdbb039-56d4-427c-8a67-645d9521df36">


--------------

_after_

<img width="684" alt="Swaps are filed in descending order of price to get you the best possible trade for your tokens" src="https://github.com/user-attachments/assets/a914eecd-4bea-4a95-8713-4f7fdb57bf9f">